src/Components/map_progression/ClickModal.css

```diff
@@ -0,0 +1,45 @@
+.modal-content {
+	-webkit-animation: slide-bottom 0.5s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
```

Comment:
`create-react-app` uses autoprefixer which adds vendor prefixes automatically. You don't need to write -webkit-.